monkey/monkey_island/cc/ui/src/components/configuration-components/InfoField.js

```diff
@@ -1,9 +1,8 @@
-import ObjectField from 'react-jsonschema-form-bs4/lib/components/fields/ArrayField';
 import * as React from 'react';
 import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
 import {faInfoCircle} from '@fortawesome/free-solid-svg-icons/faInfoCircle';
 
-class FieldWithInfo extends React.Component {
+class InfoField extends React.Component {
```

Comment:
Would "InfoBox" be a better name for this? Field implies input. Also everything that uses this widget is called "info_box".